third_party/CMakeLists.txt

```diff
@@ -51,6 +51,16 @@ if (${AMBER_USE_LOCAL_VULKAN})
   add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/vulkan-headers)
 
   set(BUILD_TESTS FALSE)
+
+  # The vulkan-loader CMake file assumes that directory exists if
+  # Wayland support is to be built.
+  if(NOT EXISTS ${WAYLAND_CLIENT_INCLUDE_DIR})
+    message(STATUS "Amber: Disabling Wayland support in Vulkan-Loader")
+    set(BUILD_WSI_WAYLAND_SUPPORT OFF CACHE BOOL "" FORCE)
```

Comment:
Should we just turn it off unconditionally like we do x11?

Reply:
I was trying to be minimally invasive.  
I unconditionally turned off X11 support because of a missing include header that I didn't want to track down.

Since this is a local build of SwiftShader (at least for now), and we want it to be headless, I could see turning everything off

Reply:
I was trying to be minimally invasive.  
I unconditionally turned off X11 support because of a missing include header that I didn't want to track down.

Since this is a local build of SwiftShader (at least for now), and we want it to be headless, I could see turning everything off